core/src/main/java/me/aurium/beetle/branch/block/Block.java

```diff
@@ -0,0 +1,19 @@
+package me.aurium.beetle.branch.block;
+
+import java.util.LinkedList;
+
+public interface Block {
+
+    String getIdentifier();
+    BlockPath asSingleBlockpath();
+
+    boolean equals(Object object);
+
+    default void addLast(LinkedList<Block> list) {
```

Comment:
You don't want to require use of a LinkedList, and I also don't see the point of these methods. I suggest that you remove the addLast(LinkedList) and addFirst(LinkedList) methods

Reply:
I was thinking about that too, but the methods have a purpose. I'll make them not default and instead leave them up to the implementer? I still need a way to ensure that some blocks can choose whether they are inserted into the path or not

Reply:
You should at the least change LinkedList to a special interface designed for Blocks to add themselves to:
```java
public interface BlockList {

  void addFirst(Block block);

  void addLast(Block block);

}
```
You don't want to couple a method signature to a LinkedList

Reply:
You're right, i'll go ahead and set that up.

Reply:
Resolved